test/e2e/suite/issuers/acme/certificate/webhook.go

```diff
@@ -197,12 +198,13 @@ func listOwnedOrders(cl versioned.Interface, owner *v1.Certificate) ([]*cmacme.O
 	}
 
 	var owned []*cmacme.Order
-	for _, o := range l.Items {
+	for i := range l.Items {
+		o := &l.Items[i]
```

Comment:
nit: We might want to copy the values, so the underlying array can be gc'ed.
This is ok since it's just test code.

Reply:
I hadn't even considered it. 
I don't know how Go garbage collection works.

If there's a block of memory somewhere containing this `Order{}` object and `o` is a pointer to it which lives beyond the scope of this loop, why should that prevent the garbage collector freeing up the memory used by the other Orders in that list?

Reply:
https://forum.golangbridge.org/t/free-memory-of-slice/3713/3